tool/travis.sh

```diff
@@ -15,18 +15,18 @@ dartanalyzer --fatal-warnings \
   test/all.dart
 
 # Run the tests.
-dart -c test/all.dart
+dart --conditional_directives -c test/all.dart
 
 # Run the UI/web tests as well.
 #pub build test
 #pub run grinder:test build/test/web.html
 
 # Verify against DDC.
-pub global activate dev_compiler
-pub global run dev_compiler lib/usage_html.dart
+#pub global activate dev_compiler
+#pub global run dev_compiler lib/usage_html.dart
 
 # Measure the size of the compiled JS, for the dart:html version of the library.
-dart tool/grind.dart build
+#dart tool/grind.dart build
```

Comment:
Likely dart2js is not fully happy with conditional imports, or we've specified something incorrectly here.